edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/logs/LogsUploader.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Core.Logs
+{
+    using System.Threading.Tasks;
+
+    public interface ILogsUploader
+    {
+        Task Upload(string uri, string module, byte[] payload, LogsContentEncoding logsContentEncoding, LogsContentType logsContentType);
```

Comment:
Who is calling this API (i.e. where do the `byte[] payload` come from?

Reply:
No-one yet.. next PR.. coming up :)